requirements/make_base.py

```diff
@@ -24,9 +24,16 @@
     f.write(header)
     f.write("\n".join(dependencies))
 
+
+def as_nightly(name: str) -> str:
+    if name == "scipp":
+        return "https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"  # noqa: E501
```

Comment:
This looks like it would break easily. Can you construct at least some of the file name from the current env? (E.g. `cp38`)
Or can you just download all wheels? Or all attached files? Will there be other files than wheels?

Reply:
I don't think one can request download of multiple versions of a wheel from a `requirements.txt` file?

I could check the Python version from the current env, but that is relatively useless, since what matter is the Python version used for creating an env from the resulting requirements file, which could be different. So is there any point in doing that?

Reply:
Right. I thought this is for a manual download. But you could change it to that. I.e., download with `requests` and point `requirements.txt` at the local file.

This should be easy if 1) we only upload one file for nightly releases, 2) all pacakges use the same python version.

Reply:
I don't think we can guarantee 1)? Can we leave it as it is for now and fix it if it breaks? Since this is in a template it will be easy to change in all projects.

Reply:
Fine. But it will break in a few days when we drop 3.8 support.

Reply:
So you are suggesting we should just ensure that we run `tox -e deps` with the same version as the nightly builds, and auto generate the `cp38` bit?

Reply:
Could do. But that creates a dependency between files that don't seem to be related. But at least it would break less on its own.